wpimath/src/main/java/edu/wpi/first/math/controller/ArmFeedforward.java

```diff
@@ -115,6 +152,8 @@ public double getKa() {
    * @param accelRadPerSecSquared The acceleration setpoint.
    * @return The computed feedforward.
    */
+  @SuppressWarnings("removal")
```

Comment:
This suppression doesn't seem necessary because this function doesn't call any deprecated methods.

Reply:
👍